src/components/DeleteTodo/DeleteTodo.tsx

```diff
@@ -0,0 +1,20 @@
+interface Props {
+  todoId: number;
+  onclick: (todoId: number) => void;
+}
+
+export const DeleteTodo: React.FC<Props> = ({ todoId, onclick }) => {
+  const handleDeleteTodo = () => {
+    onclick(todoId);
+  };
```

Comment:
redundant data comes to the component actually

```suggestion
interface Props {
  onDelete: () => void;
}

export const DeleteTodo: React.FC<Props> = ({ onDelete }) => {
....
...
 onClick={onDelete }
```

and pass it like 
`
items.map((item) => <DeleteTodo onDelete={() => handleDelete(item.id)}/>)`